hack/components/bump-kubevirt-ipam-controller.sh

```diff
@@ -0,0 +1,163 @@
+#!/usr/bin/env bash
+
+set -xeo pipefail
+
+source hack/components/yaml-utils.sh
+source hack/components/git-utils.sh
+source hack/components/docker-utils.sh
+
+IPAMCLAIMS_CRD_VERSION="v0.4.0-alpha"
+CERT_MANAGER_VERSION="v1.14.4"
```

Comment:
Can we put this info at the components.yaml ?

Reply:
no, component yaml is not for external dependencies
a whole set of bumper script consume it

anyhow we are going to remove it from here and deploy it as we are deploying kubevirt
(btw it also shows it shouldn't be on components.yaml such as kubevirt)